torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -101,32 +60,113 @@ Node* createConstantMKLDNNTensorOp(Graph* g, Tensor mkldnn_tensor) {
 }
 
 bool supportedMKLDNNWeight(const Tensor& weight) {
-  if (!weight.device().is_cpu()) {
-    return false;
+  return weight.device().is_cpu() && weight.dtype() == c10::ScalarType::Float &&
+      weight.ndimension() != 0;
```

Comment:
I guess it won't make much sense for inputs to be 0-dim

Reply:
Actually, this restriction might be able to removed, since i'm adding for support for 0-dim tensors in a later PR.